grails-app/views/purchaseOrder/_showOrderItems.gsp

```diff
@@ -232,7 +232,7 @@
                 </div>
                 <div class="right">
                     <g:if test="${!order?.isPlaced()}">
-                        <g:link controller="order" action="placeOrder" id="${order?.id}" class="button validate" >
+                        <g:link controller="order" action="placeOrder" id="${order?.id}" class="button" >
```

Comment:
@jmiranda Could now the `$(".validate").click(...);` handler be removed too? I believe it is not required anymore.